features/polyExplorer/src/screens/stories/digitalGiantsStory.jsx

```diff
@@ -47,6 +51,19 @@ const DigitalGiantsStory = () => {
         ...new Set(jurisdictionLinks.map(({ target }) => target)),
     ].filter((j) => j !== "EU-GDPR");
 
+    const handleExitStory = () => {
+        history.entries[
+            history.entries.length - 2
+        ].state.storyScrollingProgress = scrollingRef.scrollTop;
```

Comment:
Uff :D

Reply:
I hate this. I tried all things and nothing seemed to work due to the history not updating the state properly etc... Need to take a deeper look into how to do this cleverly with more time on our hands

Reply:
As for how to do this more elegantly: I'd have to check if this is technically possible, but I would try to figure out how to find the most recent history entry that corresponds to the screen this code is running for (digital giants story in this case).

Reply:
Exactly, we need to write into the old history object as the onClick of the child is triggered prior to the parents' one. I will make a component out of this